app/graphql/mutations/invoices/update.rb

```diff
@@ -10,6 +10,7 @@ class Update < BaseMutation
 
       argument :id, ID, required: true
       argument :payment_status, Types::Invoices::PaymentStatusTypeEnum, required: false
+      argument :metadata, [Types::Invoices::Metadata::Input], required: false
```

Comment:
GraphQL/ExtractInputType: Consider moving arguments to a new input type

Reply:
@vincent-pochet What do you think about this one?

Reply:
Well, if it's a best practice for GraphQL, I think it makes sense or a least it worth a try.

We could define a `UpdateInvoiceInput` type in `/app/graphql/types/invoices/update_invoice_type.rb`.
We would have to make sure we are setting `graphql_name 'UpdateInvoiceInput'` to prevent breaking the front end

What do you think?